package.json

```diff
@@ -39,6 +39,7 @@
     "pg-query-stream": "^4.2.3",
     "warp-contracts": "1.2.13",
     "warp-contracts-pubsub": "^1.0.3",
+    "warp-signature": "1.0.2",
```

Comment:
can be installed only after upgrading `warp-signature` to 1.0.3 (still waiting for the lib publication)

Reply:
btw. please move it to the new plugins monnorepo. Just remember, that for now the whole monorepo share the same tagging scheme, so the versioning must be "global" for all of the plugins.

Also, remember to set the warp-contracts in the plugin itself as a peerDependency.

Reply:
done